src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -25,41 +25,41 @@
 namespace Orleans.Serialization
 {
     /// <summary>
-    /// SerializationManager to oversee the Orleans serializer system.
+    /// The delegate used to set fields in value types.
     /// </summary>
-    public sealed class SerializationManager : IDisposable
-    {
-        /// <summary>
-        /// Deep copier function.
-        /// </summary>
-        /// <param name="original">Original object to be deep copied.</param>
-        /// <param name="context">The serialization context.</param>
-        /// <returns>Deep copy of the original object.</returns>
-        public delegate object DeepCopier(object original, ICopyContext context);
+    /// <typeparam name="TDeclaring">The declaring type of the field.</typeparam>
+    /// <typeparam name="TField">The field type.</typeparam>
+    /// <param name="instance">The instance having its field set.</param>
+    /// <param name="value">The value being set.</param>
+    public delegate void ValueTypeSetter<TDeclaring, in TField>(ref TDeclaring instance, TField value);
```

Comment:
Move to FieldUtils?

Reply:
You mean inside the class `FieldUtils`? Should I move only this one or all the delegate?

Reply:
Just this one, since it's specific to FieldUtils